schemas/IDLs/org.opencb.biodata.models/1.3.0-SNAPSHOT/avro/variantLegacy.avdl

```diff
@@ -0,0 +1,55 @@
+
+@namespace("org.opencb.biodata.models.variant.avro.legacy")
+
+protocol VariantsLegacy {
+
+    record VcfHeader {
+        string fileFormat;
+    //map<union { string, array<map<string> }>> meta; ??
```

Comment:
can we delete this line rather than commenting out ?

Reply:
same as above